sdk/examples/portal/find.ts

```diff
@@ -0,0 +1,5 @@
+import VeraxSdk from "../../src/VeraxSdk";
```

Comment:
should be removed

Reply:
Don't we want an example to find a `Portal`?

Reply:
Renamed to `findOneById``